### PR TITLE
forbid insert vertex when vertex_key flag is off

### PR DESCRIPTION
**Pull request by critical27**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [X] enhancement

#### Description:
As title, when vertex_key is not used, we forbid it. Otherwise, it will report succeeded, but no data is inserted.

Fix https://github.com/vesoft-inc/nebula-ent/issues/1420, it seems we can't simply prefix scan, we need to check if any tag is valid indeed. And valid means: tag schema exists and no ttl expired.

## Checklist:
Tests:
- [X] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Comment by critical27** [comment]
Don't merge yet, there is another tagless related issue

**Comment by critical27** [comment]
Ready to review now

File: tests/tck/features/ttl/TTL.feature [hunk 1]
```diff
@@ -394,21 +394,18 @@ Feature: TTLTest
       """
     Then the result should be, in any order, with relax comparison:
       | node  |
-      | ("1") |
```
Comment (Shylock-Hg) [inline tests/tck/features/ttl/TTL.feature:397]: Why got empty set when vertex inserted.

Comment (critical27) [reply tests/tck/features/ttl/TTL.feature:397]: It has ttl column, and ttl expired

**Review by Shylock-Hg — APPROVED** [review]

**Review by nevermore3 — APPROVED** [review]

**Pull request merged**